### PR TITLE
test(assets): Re-serialize all ORT result files

### PR DESCRIPTION
**Pull request by fviernau**
The file format meanwhile has changed slightly, so update the files by
just de-serializing and serializing them again. This also fixes broken
formatting in several places. Most importantly it simplifies upcoming
changes.

Note: The re-ordering of license IDs within SPDX expressions requires
to align the expected output files of the evaluated model reporter
tests.

### CODE REVIEW

File: advisor/src/test/assets/ort-analyzer-result.yml [hunk 1]
```diff
@@ -42,8 +42,8 @@ analyzer:
     ort_version: "HEAD"
     java_version: "11.0.9"
     os: "Linux"
-    processors: "4"
-    max_memory: "8192"
+    processors: 4
```
Comment (sschuberth) [inline advisor/src/test/assets/ort-analyzer-result.yml:45]: Commit message: Maybe better explicitly say "ORT *result* files", as this doesn't reserialize e.g. serialized `ProjectAnalyerResult`s, IIUC.

Typo: s/algin/align/

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

**Comment by sschuberth** [comment]
> @fviernau closed this 9 minutes ago

I guess something went wrong...

**Comment by sschuberth** [comment]
> > @fviernau closed this 9 minutes ago
> 
> I guess something went wrong...

Ok, somehow this still laded in `main`.

**Comment by fviernau** [comment]
> Ok, somehow this still laded in `main`.

Yeah, the UI I was giving me the error message that it could not rebase. So, I did it manually and pushed.
But when the push arrive the PR meanwhile got merged and closed. 

So all went fine in the end

**Pull request closed without merging**